### PR TITLE
Add italian and explicit support Python 3.7

### PR DESCRIPTION
**Pull request by jacopofar**
We added explicit support for Python 3.7 and for the Italian natural language using a MIT-licensed stopword list.

Python 3.3 reached EOL and is not supported by `pytest` and `pytest-cov`, so I dropped it from test targets.

Also, I added to the readme a short explanation about running tests locally on a virtualenv.

### CODE REVIEW

**Comment by jacopofar** [comment]
The CI pipeline fails for a random network error within Travis build, restarting it manually it works

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # https://blog.ionelmc.ro/presentations/packaging/
 setup(
     name="sumy",
-    version="0.7.0",
+    version="0.8.0",
```
Comment (miso-belica) [inline setup.py]: Please don't do this. I have automatic script for bumping the version on all places. Please use interactive rebacse to remove it from here and also the next commits that is manipulating it.

File: sumy/data/stopwords/italian.txt [hunk 2]
```diff
@@ -0,0 +1,656 @@
...
+giu
+già
+giã
+gli
+gliela
+gliele
+glieli
+glielo
+gliene
+governo
```
Comment (miso-belica) [inline sumy/data/stopwords/italian.txt]: I am not sure by these stop words. I always try to add only words which does not have any meaning at all. I know there are some lists on the internet which have also words like "article", "doctor", ... but I think every user should add such words on his own domain. For example this means "government" and that does not seem like a good stop-word. For some people this and other words means something and should not be removed. Please remove all words with meaning or provide some more general list of stop words. Thanks

File: setup.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 # https://blog.ionelmc.ro/presentations/packaging/
 setup(
     name="sumy",
-    version="0.8.0",
+    version="0.9.0",
```
Comment (miso-belica) [inline setup.py]: :)

File: sumy/data/stopwords/italian.txt (hunk 2 above)
Comment (jacopofar) [reply sumy/data/stopwords/italian.txt]: I see, I guess the list was produced by word counting. I'll manually remove words like this, all the other lists I found are too short

File: setup.py (hunk 1 above)
Comment (jacopofar) [reply setup.py]: Sure! For curiosity, what do you use?

Comment (miso-belica) [reply setup.py]: [bumpversion](https://github.com/miso-belica/sumy/blob/dev/tasks.py#L30)

**Comment by jacopofar** [comment]
Applied the required changes, thanks for your time!

File: sumy/data/stopwords/italian.txt (hunk 2 above)
Comment (jacopofar) [reply sumy/data/stopwords/italian.txt]: Done

**Pull request merged**